lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': {
```

Comment:
`@rarkins` enabled auto-merge but ESLint complains about the sorting order of the new preset name:

> **ESLint failure message:**
> Expected object keys to be in natural insensitive ascending order. 'airbnb-prop-types-to-prop-types-tools' should be before 'all'.

Reply:
That's funny/annoying. Wonder if we should add an ignore so that we can keep all at the top

Reply:
I searched the ESLint docs, but I can't seem to find any block ignore syntax like: `<--- eslint-ignore-this-block --->` and then `<--- eslint-start-checking-again -->`. With Prettier you can ignore a specific portion of the code, but I don't know if ESLint has that.

We can tell ESLint to ignore the whole file, but that may cause bigger problems later.

- [ESLint docs for `sort-keys` rule](https://eslint.org/docs/latest/rules/sort-keys)
- [ESLint docs, Ignore Files](https://eslint.org/docs/latest/use/configure/ignore)

Reply:
The new syntax is:

> ## Disabling Rules
> 
> ### Using configuration comments
> 
> * **Use with Caution.** Disabling ESLint rules inline should be restricted and used only in situations with a clear and
>       valid reason for doing so. Disabling rules inline should not be the default solution to resolve linting errors.
> * **Document the Reason.** Provide a comment explaining the reason for disabling a particular rule after the `--` section of the comment. This
>       documentation should clarify why the rule is being disabled and why it is necessary in that specific situation.
> * **Temporary Solutions.** If a disable comment is added as a temporary measure to address a pressing issue, create a follow-up task to address the underlying problem adequately. This ensures that the
>       disable comment is revisited and resolved at a later stage.
> * **Code Reviews and Pair Programming.** Encourage team members to review each other's code regularly. Code reviews can help
>       identify the reasons behind disable comments and ensure that they are used appropriately.
> * **Configurations.** Whenever possible, prefer using ESLint configuration files over disable comments. Configuration
>       files allow for consistent and project-wide rule handling.
> 
> To disable rule warnings in a part of a file, use block comments in the following format:
> 
> ```js
> /* eslint-disable */
> 
> alert('foo');
> 
> /* eslint-enable */
> ```
> 
> You can also disable or enable warnings for specific rules:
> 
> ```js
> /* eslint-disable no-alert, no-console */
> 
> alert('foo');
> console.log('bar');
> 
> /* eslint-enable no-alert, no-console */
> ```
> 
> ::: warning
> `/* eslint-enable */` without any specific rules listed causes all disabled rules to be re-enabled.
> :::
> 
> To disable rule warnings in an entire file, put a `/* eslint-disable */` block comment at the top of the file:
> 
> ```js
> /* eslint-disable */
> 
> alert('foo');
> ```
> 
> You can also disable or enable specific rules for an entire file:
> 
> ```js
> /* eslint-disable no-alert */
> 
> alert('foo');
> ```
> 
> To ensure that a rule is never applied (regardless of any future enable/disable lines):
> 
> ```js
> /* eslint no-alert: "off" */
> 
> alert('foo');
> ```
> 
> To disable all rules on a specific line, use a line or block comment in one of the following formats:
> 
> ```js
> alert('foo'); // eslint-disable-line
> 
> // eslint-disable-next-line
> alert('foo');
> 
> /* eslint-disable-next-line */
> alert('foo');
> 
> alert('foo'); /* eslint-disable-line */
> ```
> 
> To disable a specific rule on a specific line:
> 
> ```js
> alert('foo'); // eslint-disable-line no-alert
> 
> // eslint-disable-next-line no-alert
> alert('foo');
> 
> alert('foo'); /* eslint-disable-line no-alert */
> 
> /* eslint-disable-next-line no-alert */
> alert('foo');
> ```
> 
> To disable multiple rules on a specific line:
> 
> ```js
> alert('foo'); // eslint-disable-line no-alert, quotes, semi
> 
> // eslint-disable-next-line no-alert, quotes, semi
> alert('foo');
> 
> alert('foo'); /* eslint-disable-line no-alert, quotes, semi */
> 
> /* eslint-disable-next-line no-alert, quotes, semi */
> alert('foo');
> 
> /* eslint-disable-next-line
>   no-alert,
>   quotes,
>   semi
> */
> alert('foo');
> ```
> 
> All of the above methods also work for plugin rules. For example, to disable `eslint-plugin-example`'s `rule-name` rule, combine the plugin's name (`example`) and the rule's name (`rule-name`) into `example/rule-name`:
> 
> ```js
> foo(); // eslint-disable-line example/rule-name
> foo(); /* eslint-disable-line example/rule-name */
> ```
> 
> ::: tip
> Comments that disable warnings for a portion of a file tell ESLint not to report rule violations for the disabled code. ESLint still parses the entire file, however, so disabled code still needs to be syntactically valid JavaScript.
> :::

[ESLint docs, Disabling rules](https://eslint.org/docs/latest/use/configure/rules#disabling-rules)

Reply:
Maybe just a disable next line of possible?